h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -793,6 +795,7 @@ private void setCommonModelBuilderParams(Model.Parameters params) {
 
     params._keep_cross_validation_models = buildSpec.build_control.keep_cross_validation_models;
     params._keep_cross_validation_fold_assignment = buildSpec.build_control.nfolds != 0 && buildSpec.build_control.keep_cross_validation_fold_assignment;
+    params._export_checkpoints_dir = buildSpec.build_control.export_checkpoints_dir;
```

Comment:
for Stacked Ensembles, this will probably be ignored. If we want this to apply to SEs as well, the property needs to be propagated to the metalearner in the stack() method